docker-for-windows/faqs.md

```diff
@@ -201,36 +177,33 @@ To learn more about the reasons for this limitation, see the following discussio
 
 ### How do I add custom CA certificates?
 
-Starting with Docker Desktop for Windows 1.12.1, 2016-09-16 (Stable) and Beta 26
+Starting with Docker Desktop 1.12.1, 2016-09-16 (Stable) and Beta 26
 (2016-09-14 1.12.1-beta26), all trusted Certificate Authorities (CA) (root or
 intermediate) are supported. Docker recognizes certs stored under Trust Root
 Certification Authorities or Intermediate Certification Authorities.
 
-Docker Desktop for Windows creates a certificate bundle of all user-trusted CAs based on
-the Windows certificate store, and appends it to Moby trusted certificates. So
-if an enterprise SSL certificate is trusted by the user on the host, it is
-trusted by Docker Desktop for Windows.
+Docker Desktop creates a certificate bundle of all user-trusted CAs based on
+the Windows certificate store, and appends it to Moby trusted certificates. Therefore, if an enterprise SSL certificate is trusted by the user on the host, it is trusted by Docker Desktop.
 
 To learn more about how to install a CA root certificate for the registry, see
 [Verify repository client with certificates](/engine/security/certificates)
 in the Docker Engine topics.
 
 ### How do I add client certificates?
 
-Starting with Docker Desktop for Windows 17.06.0-ce, you do not need to push your
-certificates with `git` commands anymore. You can put your client certificates
+Starting with Docker Desktop for Windows 17.06.0-ce, you do not need to push your certificates with `git` commands anymore. You can put your client certificates
```

Comment:
Should this be "Docker Desktop?"

Reply:
I believe the product was called `Docker Desktop for Windows` in 17.06.0-ce. 

As this is an older release, I believe it is best to remove the first statement as it doesn't add much value anymore.

Reply:
Fixed in commit e055fb79.